src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -31,10 +31,36 @@ namespace Microsoft.ML.Transforms.Text
     /// </summary>
     public interface IStopWordsRemoverOptions { }
 
-    // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
-    // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
-    // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextFeaturizingEstimator "]/*' />
+    /// <summary>
+    ///  A transform that turns a collection of text documents into numerical feature vectors.
+    ///  The feature vectors are normalized counts of word and/or character ngrams (based on the options supplied) in a given tokenized text.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes. |
+    /// | Input column data type | [text](xref:System.ReadOnlyMemory<char[]>) |
+    /// | Output column data type | vector of <xref:System.Single> |
+    ///
+    /// ]]>
+    ///
+    /// This estimator gives the user one-stop solution for doing:
+    /// * Language Detection
+    /// * [Tokenzation](https://en.wikipedia.org/wiki/Lexical_analysis#Tokenization)
+    /// * [Text normalization](https://en.wikipedia.org/wiki/Text_normalization)
+    /// * [Predefined and custom stopwords removal](https://en.wikipedia.org/wiki/Stop_words)
+    /// * [Word-based or character-based Ngram and SkipGram extraction](https://en.wikipedia.org/wiki/N-gram)
```

Comment:
Does it actually produce skip-grams?

Reply:
let me error on the safe side and remove it. 

---
In reply to: [277058416](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277058416) [](ancestors = 277058416)

Reply:
It does, it's one of the advanced options for the `WordBagEstiamator.Options` that can be set in the `TextFeaturizingEstimator.Options`. The option is `SkipLength` 

---
In reply to: [277111709](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277111709) [](ancestors = 277111709,277058416)

Reply:
It would be great if you could add it back!

---
In reply to: [277113837](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277113837) [](ancestors = 277113837,277111709,277058416)